content/_data/changelogs/weekly.yml

```diff
@@ -9625,6 +9625,56 @@
   # pull: 5085 (PR title: Removed outdated browser workarounds)
   # pull: 5086 (PR title: Fixed some stylelint issues)
 
+- version: '2.270'
+  date: 2020-12-08
+  changes:
```

Comment:
also needs https://github.com/jenkinsci/jenkins/pull/5051

Reply:
Thanks!  Included in 44ae6bd676cc939b6bca968c027e4dd51e496002